test/Microsoft.IdentityModel.TestUtils/Microsoft.IdentityModel.TestUtils.csproj

```diff
@@ -24,9 +24,16 @@
     <ProjectReference Include="..\..\src\System.IdentityModel.Tokens.Jwt\System.IdentityModel.Tokens.Jwt.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
```

Comment:
Maybe you can just update this to 4.0.0?
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/fa2d252b720c777398ac5aacbf26114b053938ab/build/dependenciesTest.props#L17